.github/workflows/release-matrix.yaml

```diff
@@ -14,10 +14,6 @@ on:
         required: true
         type: string
         default: v1.30.5+k3s1
-      runner_template:
-        description: Runner template to use
-        default: hosted-prov-e2e-ci-runner-spot-n2-highmem-16-gl-template-v3
-        type: string
       operator_nightly_chart:
```

Comment:
Perhaps we can remove this input instead of `runner_template` in this file

Reply:
Why do you suggest that? I think `operator_nightly_chart` will be more useful than `runner_template` since this is a release-matrix workflow.

Reply:
Since its **release-matrix** workflow for validating operator chart rc/released versions, nightly chart might not used often

Reply:
Okay, but it can still be used to run the tests on all 3 providers while working on an automation or running suite tests, as opposed to `runner_template`, which we rarely use.